Functions/Quantification/xASL_quant_CBF.m

```diff
@@ -292,18 +298,48 @@
 fprintf('%s\n',' model with parameters:');
 
 if x.modules.asl.ApplyQuantification(3)
-    if isfield(x.Q,'LabelingDuration') && isfield(x.Q,'Initial_PLD')
-        switch lower(x.Q.LabelingType)
-            case 'pasl'
-                fprintf('%s',['TI1 = ' xASL_num2str(x.Q.LabelingDuration) ' ms, ']);
-                fprintf('%s',['TI (ms) = ' xASL_num2str(x.Q.Initial_PLD)]);
-            case 'casl'
-                fprintf('%s',['LabelingDuration = ' xASL_num2str(x.Q.LabelingDuration) ' ms, ']);
-                fprintf('%s',['PLD (ms) = ' xASL_num2str(x.Q.Initial_PLD)]);
-        end
-    else
-        fprintf('Labeling duration and initial PLD undefined...\n');
-    end
+	if modules.asl.bMultiPLD
```

Comment:
Can  you rewrite this part by treating this more similar for both multi- and single-PLD? In both cases, if you have multiple LabDur or multiple PLD you would do the same, only at the end you need a warning if singlePLD was selected but multiple PLD or LabDur detected right?

This would make the code more readable as well. Because the handling of multiple PLDs/LabDur is always the same. At the end before we quantify, we do all checks at a single place, including "warning: multiple PLDs or LabDur detected but singlePLD requested, choosing the max ... instead, this might be suboptimal"

Reply:
Very good point actually. There wasn't any reason here to separate single- and multi-PLD :) Rewritten and simplified here b0758c10